components/display/nextion.rst

```diff
@@ -47,14 +47,33 @@ Configuration variables:
 
 .. _display-nextion_lambda:
 
+Creating a HMI file
+-------------------
+
+First, you need to use the `Nextion Editor <https://nextion.itead.cc/resources/download/nextion-editor/>`__ to
+create a display file. When you open up the editor create new project and set your device type (Device model is listed at the back of the display). 
+
+Now in ``Preinitialize Event`` You have to type ``bkcmd=3``
+
+.. figure:: images/nextion-HMI1.jpg
```

Comment:
This image does not exist?

Reply:
it says you don't have permissions to pulish images. since we are making ``bkcmd=3`` publish at start we won't need that.

Reply:
here is the image that I want to display
![nextion_editor_2019-02-26_16-36-36](https://user-images.githubusercontent.com/39784853/53448322-21456200-39e5-11e9-8ad5-59b41afb6d01.png)